src/services/__test__/qr.service.test.ts

```
@@ -0,0 +1,122 @@
+jest.mock('@/utils/key.util', () => ({
+  getKeyByGroup: () => validKey,
+}));
+
+jest.mock('@/repositories/user.repository');
+
+describe('UserService', () => {
+  let service: UserService;
+  let repo: jest.Mocked<UserRepository>;
+
+  beforeEach(() => {
+    const mockPool = {} as jest.Mocked<Pool>;
```

ESLint

[error] Always prefer const x: T = { ... }.

(@typescript-eslint/consistent-type-assertions)